nodejs/eks/nodegroup.ts

```diff
@@ -626,3 +624,83 @@ function tagsToAsgTags(tagsInput: InputTags): pulumi.Output<string> {
         return output;
     });
 }
+
+/**
+ * ManagedNodeGroupOptions describes the configuration options accepted by an
+ * EKS Managed NodeGroup.
+ *
+ * See for more details:
+ * https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html
+ */
+type ManagedNodeGroupOptions = Omit<aws.eks.NodeGroupArgs, "clusterName" | "subnetIds" | "scalingConfig"> & {
+    /**
+     * The target EKS cluster.
+     */
+    cluster: Cluster | CoreData;
+
+    /**
+     * Make clusterName optional, since the cluster is required and it contains it.
+     */
+    clusterName?: pulumi.Output<string>;
+
+    /**
+     * Make subnetIds optional, since the cluster is required and it contains it.
+     *
+     * Default subnetIds is chosen from the following list, in order, if
+     * subnetIds arg is not set:
+     *   - core.subnetIds
+     *   - core.privateIds
+     *   - core.publicSublicSubnetIds
+     */
+    subnetIds?: pulumi.Output<pulumi.Output<string>[]>;
```

Comment:
It's odd that these inputs are of type `pulumi.Output` - are these intended to be `pulumi.Input`?

Reply:
Done